Training/TPU-Trillium/Llama3-405B-PyTorch/env.sh

```diff
@@ -0,0 +1,19 @@
+# Uncomment below to set the Huggingface token
+HF_TOKEN=hf_***
+PJRT_DEVICE=TPU
+XLA_IR_DEBUG=1
+XLA_HLO_DEBUG=1
+PROFILE_EPOCH=0
+PROFILE_STEP=10
+PROFILE_DURATION_MS=240000
+PROFILE_LOGDIR=/tmp/home/profile
```

Comment:
are tensorboard artifacts + tpu-info installed in this docker? @tengyifei

Reply:
tpu-info is installed. However, this training command doesn't produce any tensorboard artifacts. Even if someone installed tensorboard, they won't get any outputs. I'll look into what's needed to get those.

Reply:
are you saying profiling is not operational in this docker? if so, we can postpone tensorboard installation to the next docker and move fwd.